app/helpers/patient_helper.rb

```diff
@@ -252,70 +252,81 @@ def normalize_and_get_state_name(name)
   end
 
   def timezone_for_state(name)
+    # Offsets are DST
     timezones = {
-      'alabama' => '-05:00',
-      'alaska' => '-08:00',
-      'americansamoa' => '-11:00',
-      'arizona' => '-07:00',
-      'arkansas' => '-05:00',
-      'california' => '-07:00',
-      'colorado' => '-06:00',
-      'connecticut' => '-04:00',
-      'delaware' => '-04:00',
-      'districtofcolumbia' => '-04:00',
-      'federatedstatesofmicronesia' => '+11:00',
-      'florida' => '-04:00',
-      'georgia' => '-04:00',
-      'guam' => '+10:00',
-      'hawaii' => '-10:00',
-      'idaho' => '-06:00',
-      'illinois' => '-05:00',
-      'indiana' => '-04:00',
-      'iowa' => '-05:00',
-      'kansas' => '-05:00',
-      'kentucky' => '-04:00',
-      'louisiana' => '-05:00',
-      'maine' => '-04:00',
-      'marshallislands' => '+12:00',
-      'maryland' => '-04:00',
-      'massachusetts' => '-04:00',
-      'michigan' => '-04:00',
-      'minnesota' => '-05:00',
-      'mississippi' => '-05:00',
-      'missouri' => '-05:00',
-      'montana' => '-06:00',
-      'nebraska' => '-05:00',
-      'nevada' => '-07:00',
-      'newhampshire' => '-04:00',
-      'newjersey' => '-04:00',
-      'newmexico' => '-06:00',
-      'newyork' => '-04:00',
-      'northcarolina' => '-04:00',
-      'northdakota' => '-05:00',
-      'northernmarianaislands' => '+10:00',
-      'ohio' => '-04:00',
-      'oklahoma' => '-05:00',
-      'oregon' => '-07:00',
-      'palau' => '+09:00',
-      'pennsylvania' => '-04:00',
-      'puertorico' => '-04:00',
-      'rhodeisland' => '-04:00',
-      'southcarolina' => '-04:00',
-      'southdakota' => '-05:00',
-      'tennessee' => '-05:00',
-      'texas' => '-05:00',
-      'utah' => '-06:00',
-      'vermont' => '-04:00',
-      'virginislands' => '-04:00',
-      'virginia' => '-04:00',
-      'washington' => '-07:00',
-      'westvirginia' => '-04:00',
-      'wisconsin' => '-05:00',
-      'wyoming' => '-06:00',
-      nil => '-04:00',
-      '' => '-04:00'
+      'alabama': { offset: -5, observes_dst: true },
+      'alaska': { offset: -8, observes_dst: true },
+      'americansamoa': { offset: -11, observes_dst: false },
+      'arizona': { offset: -7, observes_dst: true },
```

Comment:
So apparently Arizona doesn't really observe DST (except for the Navajo, who do observe daylight saving time on tribal lands). I checked using Time.now.in_time_zone('Arizona').dst? as well which is currently false. Out of curiosity, what did you use to set this bool for each of the locations?

Reply:
Wow, great catch - TIL. I used this site: https://www.timeanddate.com/time/zone/guam

Pushing up a fix now.